src/components/NoticeBox.js

```diff
@@ -1,17 +1,64 @@
 import React from "react";
 
 export function NoticeBox(props) {
-  if (!props.notices) {
+  let notices = [];
+
+  // check data is valid
+  if (props.hasValidData == false) {
+    notices.push([
+      "We weren’t able to properly locate your content for Search",
+      "noticeBoxRed",
+    ]);
+  }
+
+  // check site has been indexed
+  if (props.hasBeenIndexed == false) {
+    notices.push(["Your content has not yet been indexed for Search"]);
+  }
+
+  // check at least one indexable item
+  if (props.hasItems == false) {
+    notices.push([
+      "We weren’t able to locate any content to Search to index. Perhaps you don't yet have any posts or pages?",
+    ]);
+  }
+
+  // check if current indexed items is over, their plan limit
+  // note: this currently hard codes in the number of records for the next tier.
+  // will need to be updated once this plan data is fetchable via API
+
+  if (props.recordCount > props.planRecordLimit) {
+    notices.push([
+      "You recently surpassed " +
+        props.planRecordLimit +
+        " records and will be automatically upgraded to the next billing tier of " +
+        props.planRecordLimit * 10 +
```

Comment:
This could be fragile if we change the structure of tier record limits in the future - maybe we should have the API return this too?

Reply:
yes, certainly. I hadn't quite figured out how to get the next tier in there yet - having it returned via API would be an ideal approach. I will look into that as it seems the best option here!